pykumo/py_kumo_indoor_unit.py

```diff
@@ -1,96 +1,29 @@
-""" Module to interact with Mitsubishi KumoCloud devices via their local API.
+""" Class used to represent indoor units
 """
 
 import hashlib
 import base64
 import time
 import logging
-import re
 import requests
 from urllib3.util.retry import Retry
 from requests.adapters import HTTPAdapter
 from requests.exceptions import Timeout
 from getpass import getpass
+from .const import CACHE_INTERVAL_SECONDS
+from .py_kumo import PyKumo
 
 _LOGGER = logging.getLogger(__name__)
 
-CACHE_INTERVAL_SECONDS = 5
-W_PARAM = bytearray.fromhex('44c73283b498d432ff25f5c8e06a016aef931e68f0a00ea710e36e6338fb22db')
-S_PARAM = 0
-UNIT_CONNECT_TIMEOUT_SECONDS = 1.2
-UNIT_RESPONSE_TIMEOUT_SECONDS = 8.0
-KUMO_CONNECT_TIMEOUT_SECONDS = 5
-KUMO_RESPONSE_TIMEOUT_SECONDS = 60
-
-
-class PyKumo:
+class PyKumoIndoorUnit(PyKumo):
```

Comment:
I won't really have time to look at this until the weekend, but just a quick comment: 

The issue with renaming this class is it's a breaking change. For a library with a published API, we either need to maintain backward compatibility or (less preferable) bump the major version number. There are (a handful of) additional users of pykumo library beyond HomeAssistant so just making compatible changes on both ends is not sufficient.

Reply:
Good call. I can leave that name alone and make the new top level PyKumoBase or something like that

Reply:
Would you want to bump the major anyways, since this will now expose a new class for the Kumo Station that doesn't have the same methods available to it? I've gone ahead and reverted the rename for easier backwards compatibility